include/z64camera.h

```diff
@@ -340,13 +340,13 @@ typedef enum {
 } CameraFuncType;
 
 typedef enum {
-    /* 0x00 */ CAM_DATA_00,
+    /* 0x00 */ CAM_DATA_Y_OFFSET,
     /* 0x01 */ CAM_DATA_01,
     /* 0x02 */ CAM_DATA_02,
-    /* 0x03 */ CAM_DATA_03,
+    /* 0x03 */ CAM_DATA_PITCHTARGET,
```

Comment:
```suggestion
    /* 0x03 */ CAM_DATA_PITCH_TARGET,
```
Same for `YAWDIFFRANGE`